texastribune/files
# Add up level button

#### What's this PR do?

Adds an "up one level" button to the nav.

| Old | New |
| ------------- | ------------- |
|  ![image](https://user-images.githubusercontent.com/37225902/190664568-bed9cd01-398d-438c-b0c2-486ad3494fbc.png) | ![image](https://user-images.githubusercontent.com/37225902/190664770-43c57e6e-08f6-42f1-b294-b73931b74237.png) |

#### Why are we doing this? How does it help us?

This feature has been requested by the Tribune photo dept to make it more intuitive to navigate between directories.

#### How should this be manually tested?

1) In the project, run `make start`
2) In the browser window that opens with the files app, enter a directory and then click the "up one level" button to ensure that it takes you back up a directory.
3) Test out how it looks with Tribune styles by adding the following <style> tag and contents into the <head> tag of `docs/UserInterfaceTutorial.html`:
```
    <style>
        file-browser {
            --icon-size: 25px;
            --table-row-height: 60px;
            --table-header-height: 30px;
            --search-height: 25px;
            --dropdown-icon-color: white;
            --uplevel-icon-color: white;
            --carrot-icon-color: white;
            --doc-icon-width: 60px;
            --doc-icon-height: 37px;
            --button-color: black;
            --button-hover-color: #1cacfc;
        }
    </style>
```
4) Once you've added the style tag, click save and refresh your browser window.
5) Ensure that the app looks as expected and there is no funky style bugs.

Although the test above should be sufficient, if you wish to test out _exactly_ how the app will appear/behave in the Tribune app, please follow [these instructions](https://texastribune.atlassian.net/wiki/spaces/TECH/pages/2106884097/Local+dev+with+the+Tribune+app).

#### How should this change be communicated to end users?

#### Are there any smells or added technical debt to note?

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwKqLrNFM17Lq15R/recTBGUZjsc7UJh2o?blocks=hide

#### TODOs / next steps:
